editoast/src/views/train_schedule.rs

```diff
@@ -197,8 +197,8 @@ async fn get(
 }
 
 #[derive(Debug, Deserialize, ToSchema)]
-struct BatchRequest {
-    ids: HashSet<i64>,
+pub struct BatchRequest {
+    pub ids: HashSet<i64>,
 }
```

Comment:
Maybe this `struct` needs to be moved to `src/views` since it needs to be used in multiple sub-modules.